src/polyswarm/links.py

```diff
@@ -30,3 +30,22 @@ def view(ctx, sha256):
     api = ctx.obj['api']
     output = ctx.obj['output']
     output.tag_link(api.tag_link_get(sha256))
+
+
+@link.command('list', short_help='List all links with the given tags.')
+@click.option('-t', '--tag', type=click.STRING, multiple=True,
+              help='A tag that must be associated with the artifact.')
```

Comment:
@sbnetops should each of these args be a "list" of "tags" or "families"? 

-y family -y family1
OR
-y family family1

Reply:
Multiple entries, like `-y family -y family1`

Reply:
ok. thanks